Sming/SmingCore/HardwareSerial.h

```diff
@@ -23,12 +23,13 @@ typedef Delegate<void(Stream &source, char arrivedChar, uint16_t availableCharsC
 
 class CommandExecutor;
 
-typedef struct
+struct HWSerialMemberData
 {
+	HWSerialMemberData():commandExecutor(nullptr){}
 	StreamDataReceivedDelegate HWSDelegate;
 	bool useRxBuff;
-	CommandExecutor* commandExecutor = nullptr;
```

Comment:
Reason for edit: was not sure if CommandExecutor* commandExecutor = nullptr; is sufficient, I created a constructor. If former code is OK this can be reverted.